### ISSUE TITLE
ot-br-posix failis initial installation on Rasspbian

### ISSUE THREAD

**Issue by canisLupus1313** [openthread/ot-br-posix#914]
When Following [this](https://openthread.io/guides/border-router/build) instruction installation on clean Raspbian fails.

**To Reproduce** 

1. f6c9b481004c1a0525973e10250350090319a4a5
2. Raspberry PI
3. Build steps: follow stepf 1-3 from [this](https://openthread.io/guides/border-router/build) instruction.
4. Network topology

**Expected behavior** setup script do not fail initial installation.

**Console/log output** 
```
sudo systemctl restart dnsmasq
Failed to restart dnsmasq.service: Unit dnsmasq.service is masked.
```

**Comment by jwhui** [comment]
@canisLupus1313 , can you provide more information on your setup?

I just went through the setup process and it worked well for me.

Setup:
1. ot-br-posix commit f6c9b481004c1a0525973e10250350090319a4a5
2. Raspberry Pi 4B
3. Fresh image of 2021-05-07-raspios-buster-armhf-lite.img
4. nRF52840 USB dongle with RCP image
5. Ethernet connected to Internet

Then executed the following steps on the RPi:
```
sudo apt-get update
sudo apt-get install git
git clone https://github.com/openthread/ot-br-posix
cd ot-br-posix
./script/bootstrap
INFRA_IF_NAME=eth0 ./script/setup
sudo reboot
```

**Comment by canisLupus1313** [comment]
@jwhui in my setup I'm using image full (the same release date).
And the failing command is:
`BORDER_ROUTING=0 NETWORK_MANAGER=1 ./script/setup`

**Comment by jwhui** [comment]
> @jwhui in my setup I'm using image full (the same release date).
> And the failing command is:
> `BORDER_ROUTING=0 NETWORK_MANAGER=1 ./script/setup`

@canisLupus1313 , yeah, we generally recommend to keep `BORDER_ROUTING` enabled and disable `NETWORK_MANAGER`, which is currently incompatible with the `BORDER ROUTING` feature. In fact, we were considering removing Wi-Fi AP configuration from the setup guide.

Is Wi-Fi AP on RPi functionality important for your use case?

**Comment by canisLupus1313** [comment]
I have been trying out the Android external commissioning app.
So As long as there will be working instruction and way for external commissioning I don't need RPi Wi-Fi AP.

**Comment by jwhui** [comment]
> I have been trying out the Android external commissioning app.
> So As long as there will be working instruction and way for external commissioning I don't need RPi Wi-Fi AP.

To use the latest OpenThread features, you will want to build with:
```
INFRA_IF_NAME=<netif> ./script/setup
```

Where `netif` is `wlan0` or `eth0`.

Let me know if that helps.

**Comment by canisLupus1313** [comment]
Yes this installs the services correctly and allows me to form thread network.
But how do I use external commissioning in this case Eg. android app?

**Comment by jwhui** [comment]
> Yes this installs the services correctly and allows me to form thread network.
> But how do I use external commissioning in this case Eg. android app?

You can connect RPi and mobile device to the same Wi-Fi AP.

**Comment by canisLupus1313** [comment]
Hi @jwhui sorry for delay. Yes this method works for me. And one last (I think) when and if the Wi-Fi config will be removed?

**Comment by jwhui** [comment]
> Hi @jwhui sorry for delay. Yes this method works for me. And one last (I think) when and if the Wi-Fi config will be removed?

@canisLupus1313 , we are in process of updating the docs. @Vyrastas @gavinmcgovern , thoughts on when we might be able to update docs to remove Wi-Fi AP setup?

**Comment by Vyrastas** [comment]
@jwhui We'll start on it today.

**Comment by jwhui** [comment]
> Hi @jwhui sorry for delay. Yes this method works for me. And one last (I think) when and if the Wi-Fi config will be removed?

@canisLupus1313 , https://openthread.io/guides/border-router/build has been updated to remove the Wi-Fi AP setup content. Thanks.
